src/core/src/Eryph.Configuration/Configuration/Model/CatletNetworkPortConfigModel.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Eryph.Configuration.Model
```

Comment:
Should we maybe add a comment to all models that the corresponding change handler should be updated for any changes in the model?